.github/workflows/82-kanban-slack-update.yml

```
@@ -334,6 +344,15 @@ jobs:
             $table += "$row`n"
           }
 
+          # Add the unassigned row
+          $unassignedRow = "| " + (LeftAlignText "UNASSIGNED" 17) + "|"
+          foreach ($column in $columns) {
+            $unassignedCount = $unassignedCounts[$column]
+            $leftAlignedUnassignedCount = printf "%-4s" $unassignedCount
```

```suggestion
            $leftAlignedUnassignedCount = printf "%4s" $unassignedCount
```

If we change -4 to 4, we'll get right aligned which is what I had in mind in the first place (I was mistaken before when I put -4).